packages/core/src/validate.js

```diff
@@ -263,14 +263,45 @@ export default function validateFormData(
   };
 }
 
+function isDict(v) {
+  return (
+    typeof v === "object" &&
+    v !== null &&
+    !(v instanceof Array) &&
+    !(v instanceof Date)
+  );
+}
+
+/**
+ * Get a similar schema where ref's are prefixed with "__rjsf_rootSchema"
```

Comment:
Add a comment here explaining that this is passed to ajv to represent the rootSchema in `isValid`.